src/cleanfigure.m

```diff
@@ -50,15 +50,15 @@ function cleanfigure(varargin)
 
   % Set up command line options.
   m2t.cmdOpts = m2tInputParser;
-  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'minimumPointsDistance', 1.0e-10, @isnumeric);
   m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'handle', gcf, @ishandle);
+  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'targetResolution', '15x9@600',@isstr);
```

Comment:
We can try to translate the 600 parameter into the minimum point distance and viceversa (when the user insists on using `'minimumPointsDistance'`), hence keeping backward compatibility in the syntax. 

Also, we could allow for double syntax `targetResolution` to make it easier to play with the algo:
* scalar, that sets only the points per area
* all 3 parameters in a vector, for more advanced tweaking.

For the rest I agree with @egeerardyn.